src/pages/MsigUI.vue

```diff
@@ -1,52 +1,137 @@
 <template>
-	<a-row>
-		<a-col :span="10">
-			<h1>Unsigned transaction</h1>
-		</a-col>
-		<a-col :span="4"> </a-col>
-		<a-col :span="10">
-			<h1>Signed transaction:</h1>
-		</a-col>
-	</a-row>
-	<a-row>
-		<a-col :span="10">
-			<a-textarea
-				v-model:value="unsignedJson"
-				placeholder="Basic usage"
-				:rows="4"
-				allow-clear
-			/>
-		</a-col>
-		<a-col :span="4">
-			<div>
-				<a-button type="primary" @click="sign">Sign</a-button>
-			</div>
-		</a-col>
-		<a-col :span="10">
-			<a-textarea
-				v-model:value="signedJson"
-				placeholder="Basic usage"
-				:rows="4"
-			/>
-		</a-col>
-	</a-row>
+	<a-layout-content class="contentSign">
+		<a-row>
+			<a-col :span="10">
+				<h1>Unsigned transaction</h1>
+			</a-col>
+			<a-col :span="2"> </a-col>
+			<a-col :span="12">
+				<h1>Signed transaction:</h1>
+			</a-col>
+		</a-row>
+		<a-row>
+			<a-col :span="10">
+				<div class="SignField">
+					<a-textarea
+						v-model:value="unsignedJson"
+						placeholder="Base64 or JSON object"
+						:rows="20"
+						allow-clear
+					/>
+				</div>
+				<a-button type="primary" @click="sign">SIGN</a-button>
+				<div class="switchButton">
+					<a-switch
+						v-model:checked="inputBase64"
+						checked-children="MSGPACK"
+						un-checked-children="JSON"
+					/>
+				</div>
```

Comment:
I think we should add a switch at top, below the Unsigned Transaction heading

Reply:
Sure let me change it